README.md

```diff
@@ -220,7 +230,10 @@ docker push logzio/jmx2graphite
 
 
 # Changelog
-
+- v1.1.0
+  - Major refactoring - Supporting different agents but jolokia
```

Comment:
Major refactoring - jmx2graphite now comes in two flavor: standalone using docker as it was in 1.0.x, and as a Java Agent running alongside you app. This was done to support apps running using Docker in Mesos